### DISCUSSION TITLE
SEB configuration resource not supported

### DISCUSSION THREAD

**Discussion by shareen24** [SafeExamBrowser/seb-win-refactoring#389] [Q&A]
Hi, I get this error after installing SEB. I can install it just fine but whenever I try to launch it, these messages show up. What should I do? Thanks.

<img width="328" alt="error message 1" src="https://user-images.githubusercontent.com/103158807/162105253-1fb1e80d-3c87-4b76-a725-d2580c8030b4.PNG">
<img width="442" alt="error message 2" src="https://user-images.githubusercontent.com/103158807/162105262-661148a2-5b2d-4046-8e2f-4bd7fb6f14a8.PNG">

**Comment by dbuechel** [comment]
Could you please post the log file mentioned in the second error message?

**Reply by Viditha2007** [reply]
SEB configuration resource not supported 
[DocScanner 01-Dec-2024 3-51 pm.pdf](https://github.com/user-attachments/files/17968523/DocScanner.01-Dec-2024.3-51.pm.pdf)

**Comment by danschlet** [comment]
Don't know if that's the reason for this issue, but usually a sebs:// link should be used, which will perform the configuration file download with https. seb:// links use insecure http://, which may be blocked on many systems nowadays.

**Comment by Hemalalam24** [comment]
![IMG-20250307-WA0144](https://github.com/user-attachments/assets/bf8e6fee-599c-4c7a-ae61-242ae3d456b3)

**Reply by Hemalalam24** [reply]
I have this issue please fix this

**Comment by herkules1234** [comment]
<img width="620" height="347" alt="Screenshot 2025-08-20 074634 seb error" src="https://github.com/user-attachments/assets/cd641557-790a-497c-8c1f-ddbcc71d575e" />
please help me...!
